### DISCUSSION TITLE
Domotica e antifurti Combivox e HA

### DISCUSSION THREAD

**Discussion by riccionese** [indomus/forum#931] [GENERALE - Discussioni generiche]
aloha!
c'è qualcuno che conosce **_COMBIVOX_** e magari sa se/come sia integrabile on HA ?
Ho provato a cercare sia "qui" sia in giro ma non ho trovato nulla su questo brand.
Ho amici che hanno loro prodotti e applicazioni in casa loro e non sono per niente scontenti ma gli piacerebbe poter avere il "nostro" grado di libertà di controllo/gestione/automazione/ecc. ecc. e di ampliamento anche con dispositivi di altre marche

**Comment by Maspar79** [comment]
Ciao Riccionese installo sistemi combivox per lavoro, a casa ho integrato il mio sistema in HA, purtroppo non è presente nessuna integrazione ufficiale, io ho usato un nodemcu.

**Reply by riccionese** [reply]
grazie per la risposta
quindi il nodemcu si può "attaccare" al combivox  e verificare/comandare stato dell'allarme ? (e lo stato dei vari sensori?)

**Reply by Maspar79** [reply]
[Progetto_Allarme_2.pdf](https://github.com/indomus/forum/files/7407591/Progetto_Allarme_2.pdf)
Si ho interfacciato il nodemcu alla centrale, poi tramite tasmota ho configurato tutto. Adesso sono al lavoro in serata se vuoi ti mando il package intanto allego il mio schema elettrico.
Nel mio caso avendo una centrale ibrida (sensori filari e radio) ho preferito gestire solo inserimento/disinserimento, stato allarme e allarme.

**Reply by riccionese** [reply]
Interessante. 
Credo che il fatto che tu sia un installatore di questi impianti sia determinante: un "cliente" potrebbe in autonomia aprire la centralina e fare questi collegamenti ?  Posso chiedere in che zona operi ... sai, nel caso decidessi per combivox ... :-) 

Ma ho un dubbio: questo tipo di soluzione copre "solo" la parte di antifurto o è applicabile anche alla domotica di combivox ?

**Reply by Maspar79** [reply]
Ciao, purtroppo non penso proprio che un cliente comune possa fare tutto ciò, oltre ad operare lato HA bisogna necessariamente intervenire lato centrale, nel mio caso ho configurato un ingresso per armare e disarmare e due uscite una per rilevare lo stato impianto e l'atra per lo stato allarme. Questa soluzione potrebbe anche interagire con altro ma si complicherebbe la situazione, combivox infatti gestisce la sua domotica non solo attraverso uscite a relè ma anche con attuatori su bus proprietario (tapparelle, gestione carichi, dimmer led etc.) fin quando si parlerebbe di semplici luci sarebbe fattibile, per altro dubito...
Detto questo posso tranquillamente affermare che la domotica combivox non mi fa impazzire più di tanto come del resto la sua app.
Ti ho fatto questa premessa perché secondo il mio modesto parere l'idea di gestire la domotica attraverso una centrale d'allarme mi pare un po fuori luogo, dover sacrificare ad esempio degli ingressi per accendere delle luci mi sembra uno spreco, Combivox cmq è un ottima centrale d'allarme, molto affidabile e abbastanza evoluta.
Nel caso hai bisogno puoi chiedere tranquillamente, io opero in provincia di Ragusa ma penso che siamo distanti, ad ogni modo oramai si può fare tutto telematicamente, pensa che ho un cliente perfino all'estero.

Buona serata

**Reply by riccionese** [reply]
Oltre a chiedere qui sul gruppo, ho anche scritto direttamente alla casa madre ed hanno risposto che "Con riferimento alla sua specifica richiesta, il nostro sistema si può interfacciare a dispositivi esterni esclusivamente con il protocollo KNX."
E il nostro HA mi risulta si interfacci con KNX .... può essere una cosa interessante. no?

**Reply by Maspar79** [reply]
Bella risposta che ti hanno dato....hai idea quanto costa? Ti allego uno screenshot con prezzo di listino esclusa Iva (costa più di una centrale ) Basterebbe solo che usassero il vecchio e buon protocollo mqtt. 
Non ho avuto la possibilità di montarne uno, ma penso che questo modulo più che interfacciarsi ad HA si interfacci ad un sistema Konnex esistente dopodiché tramite HA integri i due sistemi. 
![Screenshot_20211026_200044_com combivox combimanual](https://user-images.githubusercontent.com/63159996/138935929-55ad55a1-d1e0-4f00-923d-85f67a88253d.jpg)

**Reply by riccionese** [reply]
https://www.youtube.com/watch?v=OaOTxxF7dz8

**Comment by Maspar79** [comment]
Buon pomeriggio, rievoco questo vecchio thread, per informare a chi fosse interessato che ad oggi la Combivox ha finalmente fatto un passo avanti. Con la commercializzazione delle nuove centrali di antintrusione (non faccio il nome perchè non è mia intenzione pubblicizzarle) finalmente più consone ad una domotica proprietaria (in questi anni hanno cercato di migliorare) hanno dato seppur in modo "elementare" la possibilità di interfacciarsi ad altri sistemi.
Premesso che sono un installatore ed essendo anche un felice utilizzatore di HA, ho avuto la possibilità di "giocare" un pò con i due sistemi. La centrale può ricevere ed inviare stringhe http (con tutti i limiti del caso) sicuramente non il massimo, ma grazie all'introduzione di "Macro" (quello che per noi sono le automazioni) si possono finalmente inviare e ricevere stringhe ad altri dispositivi, in questo caso ad HA.

Ho fatto delle prove utilizzando un trigger webhook, per inviare da centrale ad HA ho creato una macro che invia una stringa, lato HA  ho creato invece un automazione che legge la stessa, viceversa invio da HA tramite automazione una stringa alla centrale, la stessa tramite macro riceve la stringa ed esegue un'azione. In conclusione senza spendere soldi si possono mettere in comunicazione i due sistemi, non perdendo comunque di vista la filosofia che un sistema di antifurto deve essere sempre e comunque gestito da un prodotto professionale, certificato da aziende del settore, non mi sognerei mai ad esempio d'inserire e disinserire l'antifurto tramite stringa http, verrebbe meno la sicurezza stessa del sistema.
